week1/1.5/Notes/Notes/Controllers/View Controllers/NoteTableViewController.swift

```diff
@@ -0,0 +1,136 @@
+//
+//  NoteTableViewController.swift
+//  Notes
+//
+//  Created by Eric Lanza on 7/6/18.
+//  Copyright © 2018 ETLanza. All rights reserved.
+//
+
+import UIKit
+
+class NoteTableViewController: UITableViewController {
+    
+    //MARK: - IBOutlets
+    @IBOutlet weak var editBarButton: UIBarButtonItem!
+    
+    //MARK: - IBActions
+    @IBAction func editButtonPressed(_ sender: UIBarButtonItem) {
+        if  tableView.isEditing {
+            tableView.setEditing(false, animated: true);
+            editBarButton.title = "Edit";
+            editBarButton.style = UIBarButtonItemStyle.plain;
+        } else {
+            tableView.setEditing(true, animated: true);
+            editBarButton.style = UIBarButtonItemStyle.done
+            editBarButton.title = "Done";
+            editBarButton.style =  UIBarButtonItemStyle.done;
+        }
+    }
+    
+    //MARK: - Properties
+    let searchController = UISearchController(searchResultsController: nil)
+    
+    var filteredNotes = [Note]()
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        NoteController.shared.loadFromPersistentStore()
+        searchController.searchResultsUpdater = self
```

Comment:
This code is kinda clustered. Structure it by white space or create a `setupSearchController` method or something.